specification/network/resource-manager/Microsoft.Network/stable/2019-02-01/expressRouteGateway.json

```diff
@@ -536,19 +536,9 @@
                     }
                 },
                 "provisioningState": {
-                    "type": "string",
                     "readOnly": true,
```

Comment:
The definition already contains "readOnly": true. This should be removed from here.